src/actinia_core/core/common/config.py

```diff
@@ -708,6 +736,26 @@ def read(self, path=DEFAULT_CONFIG_PATH):
                         "API", "ENDPOINTS_CONFIG"
                     )
 
+            if config.has_section("KEYCLOAK"):
+                if config.has_option("KEYCLOAK", "CONFIG_PATH"):
+                    keycloak_cfg_path = config.get("KEYCLOAK", "CONFIG_PATH")
+                    if os.path.isfile(keycloak_cfg_path):
+                        self.KEYCLOAK_CONFIG_PATH = keycloak_cfg_path
+                    # else:
+                    #     log.warning(
+                    #         f"Keycloak is configured, but configfile is not "
+                    #         "an existing file! Using Redis for user "
+                    #         "management."
+                    #     )
```

Comment:
Is this supposed to be activated later? If yes, please add a TODO, if not, please remove.

Reply:
It is not possible to activate it because then I get an circle of imports. But I would find it good if this would be logged somewhere. Ideas are very welcome

Reply:
Maybe use a simple `print` statement? Not so nice as it cannot be configured if it is json or not, but maybe better than nothing?